src/collection/resources/_resources.js

```diff
@@ -34,13 +34,14 @@ class Components {
       apiPath = this.apiPath,
       action = '',
       ResConstructor = this.ResConstructor,
+      isJson = true,
       json,
```

Comment:
I'm not especially happy with this naming, as we now have an `isJson` and a `json` parameter in this function.
The `json` parameter allows for choosing if we would like an object or a json response from the promise/callback.
The `isJson` parameter allows for choosing if we should `JSON.stringify` the body before sending, and what `Content-Type` we should apply to the request.

Reply:
hm, maybe it's better to have a content_type parameter, that defaults to application/json?

Reply:
Yeah, that's what I thought, but the content type we need to send looks like this:
`multipart/form-data; boundary=---------------------------1907295027419120020403813980`
Much easier to let superagent deal with it, no?

Reply:
I think the browser is adding the `boundary` stuff automatically if it recognizes the mutipart.